x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/index.js

```diff
@@ -151,6 +191,12 @@ export class InnerCustomPlot extends PureComponent {
             tickFormatX={this.props.tickFormatX}
           />
 
+          {stacked
+            ? React.cloneElement(staticPlot, {
```

Comment:
AFAICT, there's only one way to achieve the look that we want (stroke on just one side of the area), and that is by drawing another line chart on top of it. Seems overkill, but it's the recommendation from the [docs](https://github.com/uber/react-vis/blob/master/docs/area-series.md):
> To create a chart that has a fill, no distinct lines to the left, right or bottom, but a different line style at the top, you may create an area chart with a line chart on top.

Reply:
that's unfortunate, but tolerable as long as there's not any noticeable drag to page performance.

Reply:
@dgieselaar Doesn't this duplicate the the XAxis, YAxis and HorizontalGridLines? What about just adding an additional LineSeries to the chart?